reoptjl/views.py

```diff
@@ -734,12 +734,23 @@ def link_run_uuids_to_portfolio_uuid(request):
                 for s in scenario:
                     s.portfolio_uuid = p_uuid
                     s.save()
+
```

Comment:
@Bill-Becker Am I allowed to delete `PortfolioUnlinkedRuns` objects?

I realized that when we link a portfolio to an existing run, there can be a situation where the run is listed in `PortfolioUnlinkedRuns`, ie it was unlinked from a portfolio previously. In this case we do not want the run to show up in independent summary endpoint runs, which is why we'd have to delete that entry from `PortfolioUnlinkedRuns` table. I tested to see that an existing run can be deleted from `PortfolioUnlinkedRuns` and tied to a portfolio so it doesnt show up in independent summary runs, and then unlinked so it does show up.